apps/web/.babelrc

```diff
@@ -1,15 +1,6 @@
 {
-  "plugins": [
-    "@babel/plugin-proposal-class-properties",
-    "@babel/plugin-proposal-private-methods"
-  ],
   "presets": [
-    [
-      "@babel/preset-env",
-      {
-        "useBuiltIns": "usage",
-        "corejs": "3.21"
-      }
-    ]
+    "@babel/preset-env",
+    "@babel/preset-react"
```

Comment:
Did you mean to change the babel presets in this PR?

Reply:
Yeah, so the class properties and private methods are included in env and for some reason the packages were not working without the react preset. I didn't think we were using `core-js` cause it was not installed but I realized we had the wrong dep installed so I updated that to fix it 👍 

[re-add core-js for babel](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5697/commits/d8c9d771cf611a2b5b5bc140b46690ae426fa2d8)